### ISSUE TITLE
Byteconf Web Animation 2019

### ISSUE THREAD

**Issue by enriquezm** [byteconf/conferences#7]
### What's the name of your proposed event? For instance, "Byteconf GraphQL". 

`Byteconf Motion, Byteconf React Animation or Byteconf React Motion`

### What would the event cover? 

`Creating practical animations/motion with React based software`

### Do you have any speakers you'd like to see at the conference? 

`Matt Perry @mattgperry`

### Anything else you want us to know about your idea? 

`Usually this topic is mixed into conferences here and there, but I don't believe it gets the attention it deserves. Animation/motion within the UI is a growing concern and a lot of developers aren't sure how to implement them. Here are some articles discussing what I'm talking about in more detail: [Proper us of animation](https://uxdesign.cc/the-ultimate-guide-to-proper-use-of-animation-in-ux-10bd98614fa9), [Animation Guidelines in UX](https://blog.prototypr.io/6-animation-guidelines-for-ux-design-74c90eb5e47a).

**Comment by signalnerve** [comment]
This is a great idea. I think we could probably fill 5-6 slots here, but I am a bit worried that it's a bit specific for a full-length conference. I'm really interested, but will think on how best to do this. Open to thoughts on the format or speaker suggestions

**Comment by enriquezm** [comment]
Yeah, it may be short for a full-length conference. But just for the heck of it, here's what I think a potential rough outline for such a conference would look like:

- Animation fundamentals 
- Types of animation
    - Transitional
    - Supplemental
    - Feedback
    - Demonstrative
    - Decorative
- Accessibility and animations
- Optimizing animations 
- Animation libraries 

Please add if anyone knows more.
